server.js

```diff
@@ -1,16 +1,24 @@
 /* eslint-disable no-console */
 import express from 'express';
-import routes from './server/routes/router';
+import userRoutes from './server/routes/userRouter';

```

Comment:
You should import these two files into an `routes/index.js` file which you will then import here and use.